cmd/zetaclientd/keygen_tss.go

```diff
@@ -16,9 +17,9 @@ import (
 	"gitlab.com/thorchain/tss/go-tss/p2p"
 )
 
-func GenerateTss(logger zerolog.Logger, cfg *config.Config, zetaBridge *mc.ZetaCoreBridge, peers p2p.AddrList, priKey secp256k1.PrivKey, ts *mc.TelemetryServer) (*mc.TSS, error) {
+func GenerateTss(logger zerolog.Logger, cfg *config.Config, zetaBridge *mc.ZetaCoreBridge, peers p2p.AddrList, priKey secp256k1.PrivKey, ts *mc.TelemetryServer, tssList []*types.TSS) (*mc.TSS, error) {
```

Comment:
Do we need using array of pointer? It seems to me that we never use `nil` value?

Reply:
We don't need it , but I don't see any issues with keeping it. Do you see any issues with the pointer ?